snmp/datadog_checks/snmp/data/profiles/_base_cisco.yaml

```diff
@@ -6,6 +6,9 @@ extends:
   - _generic-router-tcp.yaml
   - _generic-router-udp.yaml
 
+tags: 
```

Comment:
I think it's `metri_tags` what you need to use https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/conf.yaml.example#L33